stable/aws-node-termination-handler-chart/README.md

```diff
@@ -0,0 +1,34 @@
+# AWS Node Termination Handler Chart
+AWS Node Termination Handler Helm chart for Kubernetes. For more information on this project see the project repo at https://github.com/aws/aws-node-termination-handler. 
+## Prerequisite
+* Kubernetes >= 1.11
+## Installing the Chart
+Add the EKS repository to Helm:
+```sh
+helm repo add eks https://aws.github.io/eks-charts
+```
+Install AWS Node Termination Handler:
+```sh
+helm upgrade -i aws-node-termination-handler eks/aws-node-termination-handler
+```
+## Uninstalling the Chart
+
+To uninstall/delete the `aws-node-termination-handler` deployment:
+
+```console
```

Comment:
consider being consistent with the blocks above and use `sh` instead of `console` here.